tests/conftest.py

```diff
@@ -53,6 +53,9 @@ def sort(self, sort):
                 self.data.reverse()
         return self.data
 
+    def collation(self, collation):
+        return self
```

Comment:
We want to handle the collation in the `sort` and then write a test for this. :wink:

Reply:
i didn't understand. How am i not going to pollute the sort if I add the collation in it ?

Reply:
I'm not sure to understand your question. All I say is that we need tests for `paginate(collation=...)`, whatever the implementation is. :slightly_smiling_face:

Reply:
@ramnes sorry for the delay, this should be ok now though.